### DISCUSSION TITLE
Why OCIRepository crd has tag/semver/diggest?

### DISCUSSION THREAD

**Discussion by mbaykara** [fluxcd/flux2#3002] [General] [answered]
Is it not better UX to remove required to give `tag` or default `latest` for OCIRepository? Because I want to specify the OCI artifact version on corresponding `Helmrelease`, otherwise for each Helmrelease that deployed with different version I need to create a `OCIRepository` crd. Do I wrong?

Thanks alot

**Comment by somtochiama** [comment]
You don't need to create new crd.

The OCIRepository has a `spec.semver` where you can specify a semver range for flux to pick from. (if you don't use semver for your tags, you can update the tag in git and Flux will apply it)

**Reply by mbaykara** [reply]
Still this is not how I expected similar to Helmrepository, but below @stefanprodan pointed, my scenario was any wrong :) Thanks

**Comment by stefanprodan** [comment, marked as answer]
OCIRepositories are not made for Helm, they can be used as sources for Flux Kustomizations only. Please see here how Helm OCI works: https://fluxcd.io/docs/cheatsheets/oci-artifacts/#helm-oci

**Reply by mbaykara** [reply]
I thought ist also for the Helmreleases as well. I was using as in link you shared before, today I tried the with OCIRepository. Ok then the problem been solved, thanks
